src/Test/Utilities/Portable/Compilation/OperationTreeVerifier.cs

```diff
@@ -904,13 +904,6 @@ public override void VisitLambdaExpression(ILambdaExpression operation)
         public override void VisitLiteralExpression(ILiteralExpression operation)
         {
             LogString(nameof(ILiteralExpression));
-
-            object value;
-            if (operation.ConstantValue.HasValue && ((value = operation.ConstantValue.Value) == null ? "null" : value.ToString()) == operation.Text)
-            {
-                LogString($" (Text: {operation.Text})");
```

Comment:
To make the change easier, you coudl just write out operation.Syntax.ToString here.